subsys/bluetooth/host/classic/l2cap_br.c

```diff
@@ -2074,9 +2076,17 @@ void bt_l2cap_br_init(void)
 		bt_avdtp_init();
 	}
 
+	if (IS_ENABLED(CONFIG_BT_AVCTP)) {
+		bt_avctp_init();
+	}
+
 	bt_sdp_init();
 
 	if (IS_ENABLED(CONFIG_BT_A2DP)) {
 		bt_a2dp_init();
 	}
+
+	if (IS_ENABLED(CONFIG_BT_AVRCP)) {
+		bt_avrcp_init();
+	}
```

Comment:
This isn't particularly scalable. It should be possible to implement new profiles/protocols completely on the application side, which means that you can't have a requirement that the stack-internal `l2cap_br.c` needs to be modified every time you add some new profile. The profiles/protocols which are part of the Bluetooth core specification (L2CAP, SDP, etc) could be treated as an exception, but for everything else you need to find a solution which doesn't require modifying this internal c-file.